### ISSUE TITLE
Cannot find module 'hyper/component'

### ISSUE THREAD

**Issue by bit-cmdr** [Hyperline/hyperline#135]
Getting an error at startup/reload

```
Plugin "hyperline" failed to load (Cannot find module 'hyper/component')
```

This just started yesterday. I tried closing and relaunching, now I have no colors for hyperline, previously each component was color-coded, and I have a Spotify icon and a Party Parrot gif. While waiting for a fix, is there any way to disable the Spotify and Party Parrot?

**Comment by MilesOkeefe** [comment]
This can be fixed by downgrading from yesterday's 1.0.0-beta.1.1 release to the previous, 0.6.1.

So just edit your ~/.hyper.js file and change

> plugins: [
> "hyperline",
> ...
> ]

to

> plugins: [
> "hyperline#0.6.1",
> ...
> ]

Not sure why this beta got pushed upstream, seems totally broken.

**Comment by bit-cmdr** [comment]
Confirmed that it does indeed fix the issue. Thanks so much!
